src/aspire/basis/steerable.py

```diff
@@ -486,3 +475,51 @@ def to_complex(self, coef):
             ind += np.size(idx)
 
         return ComplexCoef(self, complex_coef)
+
+    @abc.abstractmethod
```

Comment:
Can you elaborate on the motivation here? Is it to make sure that each subclass doesn't blindly inherit this method and checks whether it applies first?

Reply:
Yes, that is the idea. If a new subclass of SteerableBasis2D is implemented, the abc would force the method to be explicitly implemented (even if just calling super()). Hopefully this encourages new basis authors to consider this method.  Right now we have three implementations.  We both know these can be improved, but can be smaller PR(s).

I can remove and revert to pure inheritance if you prefer.  It would cut out a few lines of code in FB, PSWF, and FPSWF.  I'm not attached to the abc, sometimes it just helps spell out the interfaces/protocols.

Reply:
No that sounds like a good idea. Might just be good with a comment or so explaining why it's declared abstract (just what you wrote here).

Reply:
Added comment.

Reply:
Thanks!